### PR TITLE
LPS-191853 Remove story FF: Save ObjectEntry as a draft (LPS-181663)

### PR DESCRIPTION
**Pull request by p-albuquerque**
Related issue: https://issues.liferay.com/browse/LPS-191853

### CODE REVIEW

**Comment by marcelabc** [comment]
Hey @p-albuquerque ,
Since you are a member of the BPM team, you are expected to assign someone from the team as the reviewer.
Assigning the reviewer is a way to organize our line and make the review process agile.
If you don't know who to assign, please talk with @guilhermedcamacho .

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page_structure/components/item_configuration_panels/FragmentGeneralPanel.js [hunk 1]
```diff
@@ -87,9 +87,8 @@ export function FragmentGeneralPanel({item}) {
 					let fields = fieldSet.fields;
 
 					if (
```
Comment (pedro-oliveira446) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page_structure/components/item_configuration_panels/FragmentGeneralPanel.js:89]: Hey @p-albuquerque,
wouldn't it be better to remove the entire if block?
This was added in this commit : https://github.com/liferay-echo/liferay-portal/pull/13773/commits/82b656fe1c5663dd64fe5aab1a9ec322b148ee28.

**Review by pedro-oliveira446 — CHANGES_REQUESTED** [review]

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page_structure/components/item_configuration_panels/FragmentGeneralPanel.js (hunk 1 above)
Comment (p-albuquerque) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page_structure/components/item_configuration_panels/FragmentGeneralPanel.js]: Done

**Review by pedro-oliveira446 — APPROVED** [review]

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by guilhermedcamacho** [comment]
ci:test:object

**Comment by guilhermedcamacho** [comment]
Hey @p-albuquerque , can you please check the CI results?

**Review by guilhermedcamacho — APPROVED** [review]

**Comment by p-albuquerque** [comment]
I'm working on it

**Comment by p-albuquerque** [comment]
object suite: seems to be alright;
relevant suite: I'm analyzing some fragment test failures

**Comment by p-albuquerque** [comment]
ci:test:relevant

**Comment by p-albuquerque** [comment]
ci:test:object

**Comment by p-albuquerque** [comment]
Directly sent: https://github.com/brianchandotcom/liferay-portal/pull/145898

**Pull request closed without merging**